doc/monitoring/metrics_reference.rst

```diff
@@ -207,3 +207,77 @@ Runtime
 * ``tnt_runtime_lua`` – Lua garbage collector size in bytes
 
 * ``tnt_runtime_used`` - number of bytes used for Lua runtime
+
+.. _metrics-cartridge:
+
+-------------------------------------------------------------------------------
+Cartridge
+-------------------------------------------------------------------------------
+
+* ``cartridge_issues`` - Number of
+  `issues <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.issues/>`
+  across cluster instances. This metric always has labels - ``{level="critical"}``.
+  ``level`` - the level of the issue. ``critical`` level is critical cluster problems,
+  e.g. memory used ratio > 90%, ``warning`` is problems that should be considering,
```

Comment:
are problems

Maybe considered? I don't know how to fix this. What did you try to say?